packages/slice-machine/tests/apiController/preview.test.ts

```diff
@@ -13,7 +13,7 @@ jest.mock("@slicemachine/core/build/src/fs-utils", () => ({
 describe("preview controller", () => {
```

Comment:
Can you drop this commit ? I already did this here : https://github.com/prismicio/slice-machine/pull/272/commits/1deb9fb4d171bfbbe3998027a6936be2ecb646bd

Reply:
I had conflicts in the test and had to resolve manually